generation/WinSDK/enums.json

```diff
@@ -4375,6 +4384,18 @@
       {
         "method": "CreatePen",
         "parameter": "iStyle"
+      },
+      {
+        "struct": "LOGPEN",
+        "field": "PEN_STYLE"
```

Comment:
You've got the enum name here instead of the field name. For example on LOGPEN it's lopnStyle. I don't understand how your .txt file has an update for that field when it's not right in this file though.

Reply:
I think you must have had a previous change where this was working. I need to update the diffing tool to detect when there are lines in the exception file that aren't be used, and make it an error until they're removed.

Reply:
Good catch, and you're right - it was a from a previous change that did work. I updated the enum and changes file so it should be correct now.